### PR TITLE
Some DUARel combinators

### PR DESCRIPTION
**Pull request by ecavallo**
- Define `SubstRel` generalizing the `TransportStr` from the structure code.
- Add some (D)UARel combinators for sigma and function types and organize existing combinators in files according to type former
- Define morphisms of UARels and show that DUARels can be reindexed along these.

(There will be a follow-up PR using this stuff to generalize the auto from Structure.)

### CODE REVIEW

File: Cubical/Displayed/Universe.agda [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+open import Cubical.Displayed.Subst
+
+private
+  variable
+    ℓA ℓ≅A ℓB ℓ≅B ℓP : Level
+
+𝒮-Univ : ∀ ℓ → UARel (Type ℓ) ℓ
+𝒮-Univ ℓ .UARel._≅_ = _≃_
+𝒮-Univ ℓ .UARel.ua _ _ =
+  isoToEquiv (invIso (Univalence.isoThm pathToEquiv pathToEquivRefl))
```
Comment (mortberg) [inline Cubical/Displayed/Universe.agda:25]: You can use https://github.com/agda/cubical/blob/master/Cubical/Foundations/Univalence.agda#L207

Or take the inverse equiv of univalence?

**Review by mortberg — CHANGES_REQUESTED** [review]
Good stuff! I just had one small comment and then we can merge

File: Cubical/Displayed/Universe.agda (hunk 1 above)
Comment (ecavallo) [reply Cubical/Displayed/Universe.agda]: Ah, I didn't notice `univalenceIso`. I was avoiding inverse equiv to not do unnecessary `isoToEquiv`s, though it probably doesn't matter much here.

**Review by mortberg — APPROVED** [review]

**Comment by mortberg** [comment]
Let's merge once Travis is done

File: Cubical/Displayed/Universe.agda (hunk 1 above)
Comment (mortberg) [reply Cubical/Displayed/Universe.agda]: Right, I suspected that was the reason

**Pull request merged**